docs/src/dev/future/proposal-transaction-4.asciidoc

```diff
@@ -0,0 +1,113 @@
+////
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+////
+image::apache-tinkerpop-logo.png[width=500,link="https://tinkerpop.apache.org"]
+
+*x.y.z - Proposal 4*
+
+== Tinkerpop native transaction support
+
+=== Introduction
```

Comment:
I think there's more to be said here and in "Motivation" about this. The neo4j point has validity but there's a variety of other drivers when I think about what this feature is. Test framework simplification, a more robust Gremlin Server setup out-of-the-box, a better learning experience for new users, a less complicated transaction story for new users, a better test graph that is more readily swappable with other graphs, and probably more. Please expand these sections further. 

Also, please get "TinkerPop" spelling corrections in place. 🙂

Reply:
updated in https://github.com/apache/tinkerpop/pull/2054/commits/4f3f7e916cf4b44db550c67663bd1f4d55bac598